fw/hpack.c

```diff
@@ -1252,6 +1252,9 @@ tfw_hpack_hdr_set(TfwHPack *__restrict hp, TfwHttpReq *__restrict req,
 	case TFW_TAG_HDR_RAW:
 		parser->_hdr_tag = TFW_HTTP_HDR_RAW;
 		break;
+	case TFW_TAG_HDR_X_TEMPESTA_CACHE:
+		parser->_hdr_tag = TFW_HTTP_HDR_X_TEMPESTA_CACHE;
+		break;
```

Comment:
So far the case for the tag `TFW_TAG_HDR_RAW` was tried after all other more definitive (_special_) headers were tried. Perhaps, it's best to follow that convention.

Reply:
done